x-pack/plugins/file_upload/public/components/import_complete_view.tsx

```diff
@@ -76,7 +76,7 @@ export class ImportCompleteView extends Component<Props, {}> {
           </EuiFlexItem>
         </EuiFlexGroup>
         <div style={{ height: '200px' }}>
-          <CodeEditor
+          <CodeEditor // TODO need to provide theme in context above. Need CodeOwners help.
```

Comment:
@nreese I'm not exactly sure how to get to this screen. I think I might need to successfully upload some features first? Would you mind verifying this one and the `tile_request_tab`?

Reply:
Expose getTheme method in https://github.com/elastic/kibana/blob/main/x-pack/plugins/file_upload/public/kibana_services.ts and then use the getter function to get theme in this component

Reply:
@nreese On which screen can I find this CodeEditor? Would you mind helping me navigate again 😅

Reply:
* Download world countries geojson at https://vector.maps.elastic.co/files/world_countries_v7.geo.json?elastic_tile_service_tos=agree&my_app_name=ems-landing-page&my_app_version=8.8.1&license=643c1faf-80fc-4ab0-9323-4d9bd11f4bbc and save into a file
* Create new map
* Click "Add layer"
* Select "Upload"
* Select world countries geo json file
* Import file.
* Results displayed in code editor.

Reply:
Thanks @nreese ! I added `theme` to `kibana_services`and it's working now. See https://github.com/elastic/kibana/pull/159638/commits/abcc1700f68ca99c53d324e681bed86aa5691064
<img width="458" alt="Screenshot 2023-06-28 at 3 53 16 PM" src="https://github.com/elastic/kibana/assets/21210601/83266c69-f6d0-4f91-98f8-e82107f169ad">